metaflow/datastore/s3.py

```diff
@@ -105,7 +105,7 @@ def get_latest_tasks(cls,
                 task_urls = [task.url for task in task_objs]
             urls = []
             for task_url in task_urls:
-                for attempt in range(5):
```

Comment:
`MAX_ATTEMPTS` seems to be 6 but we used to have `range(5)`. Let's just double check that the change is intentional